org.eclipse.scanning.test/src/org/eclipse/scanning/test/BrokerTest.java

```diff
@@ -83,7 +83,7 @@ public final static void setUpNonOSGIActivemqMarshaller(Class<?>...extras) {
 
 	@AfterClass
 	public final static void stopBroker() throws Exception {

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)